src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -129,6 +133,58 @@ const CommentView = View.extend({
   },
 });
 
+const CommentView = View.extend({
+  className: 'u-margin--b-8 qa-activity-item',
+  modelEvents: {
+    'change:editing': 'render',
+  },
+  childViewTriggers: {
+    'click:edit': 'click:edit',
+    'cancel:comment': 'cancel:edit',
+    'post:comment': 'save:comment',
+    'delete:comment': 'delete:comment',
+  },
+  template: hbs`
+    <div data-display-region></div>
+    <div class="sidebar_activity--comment-form" data-form-region></div>
+  `,
+  regions: {
+    display: '[data-display-region]',
+    form: '[data-form-region]',
+  },
+  onRender() {
```

Comment:
I think the `CommentDisplayView` could possibly just have a wrapping div that's a region.. then `onClickEdit` just show the comment form in that region.. and on save or cancel re-render that view which will clear the form rather than adding data to the model.